README.md

```diff
@@ -201,7 +201,16 @@ Once you have the Compiled DB, it is then possible to use the `g2p convert` comm
 
 You can also run the `g2p Studio` which is a web interface for creating custom lookup tables to be used with g2p. To run the `g2p Studio` either visit https://g2p-studio.herokuapp.com/ or run it locally using `python run_studio.py`.
 
-Alternatively, you can run the app from the command line: `g2p run`
+## API for Developers
+
+There is also a REST API available for use in your own applications.
+To launch it from the command-line use `flask --app g2p.app run`.  The
+API documentation will be viewable (with the ability to use it
+interactively) at http://localhost:5000/docs - an OpenAPI definition
+is also available at http://localhost:5000/static/swagger.json .
```

Comment:
These two links also work when we run `python run_studio.py`. Is there an advantage to running it via `flask --app` instead of via our our script? In any case, to avoid confusion since the two commands are effectively equivalent, I would say here 

    To launch it from the command-line use `flask --app g2p.app run` or `python run_studio.py`.

so that it's clear you get the same thing either way.

Both commands actually spin up the g2p-studio on http://localhost:5000 so it'll be confusing otherwise.

Reply:
Oh, the API docs and swagger are also available on https://g2p-studio.herokuapp.com/ but I don't know if we need to mention that here.

Reply:
I figured I'd leave that out since if people are develop-y enough to use the API then maybe they can self-host...